appveyor.yml

```diff
@@ -13,7 +13,7 @@ configuration:
 before_build:
   - git submodule update --init --recursive
   - python get_steinberg_sdk.py
-  - cmake -G "Visual Studio 14" Smexoscope
+  - cmake -G "Visual Studio 14 2015 Win64" .
```

Comment:
I think the only way to do both x86 and x64 would be to have two builds. As far as I can tell Cmake has no way generating a solution with both x86 and x64 targets. You would have to call cmake, build, clean, call cmake x64 and build again. Does appveyor support something like that?